browser-interface/packages/entryPoints/index.ts

```diff
@@ -64,7 +64,7 @@ globalThis.DecentralandKernel = {
     }
 
     if (!Hls.isSupported()) {
-      throw new Error('HTTP Live Streaming is not supported in your browser')
+      // throw new Error('HTTP Live Streaming is not supported in your browser')
```

Comment:
As a hotfix I will totally remove it. And we can add a url parameter to not check it.